src/Nethermind/Nethermind.Runner/JsonRpc/JsonRpcIpcRunner.cs

```diff
@@ -166,20 +170,14 @@ private void DeleteSocketFileIfExists(string path)
 
         public void Dispose()
         {
-            if (_server != null)
-            {
-                _server.Close();
-                _server.Dispose();
-            }
-
+            _server?.Dispose();
             DeleteSocketFileIfExists(_path);
-
             if (_logger.IsInfo) _logger.Info("IPC JSON RPC service stopped");
         }
 
-        private void LogInfo(string msg)
+        private void LogDebug(string msg)
         {
-            if (_logger.IsInfo) _logger.Info(msg);
+            if (_logger.IsDebug) _logger.Debug(msg);
```

Comment:
Maybe we should build logger interface without the need of adding IsDebug, IsTrace ;) Of course, in future ;)

Reply:
its not possible, since this if is to not calculate string params, which can be costly

Reply:
ah got it, thx